cluster/gce/upgrade.sh

```diff
@@ -33,7 +33,7 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 function usage() {
   echo "!!! EXPERIMENTAL !!!"
   echo ""
-  echo "${0} [-M|-N|-P] -l -o | <version number or publication>"
+  echo "${0} [-M|-N|-P] [-o] -l | <version number or publication>"
```

Comment:
why is -o in it's own set of brackets?

also, can you update the usage for -l to print that it is only supported for upgrading the master and not nodes?